lib/internal/modules/helpers.js

```diff
@@ -298,6 +298,30 @@ function normalizeReferrerURL(referrer) {
   return new URL(referrer).href;
 }
 
+/**
+ * Determines whether a specifier is a relative path.
+ * @param {string} specifier - The specifier to check.
+ */
+function isRelativeSpecifier(specifier) {
+  if (specifier[0] === '.') {
+    if (specifier.length === 1 || specifier[1] === '/') { return true; }
+    if (specifier[1] === '.') {
+      if (specifier.length === 2 || specifier[2] === '/') { return true; }
+    }
+  }
+  return false;
+}
+
+/**
+ * Determines whether a specifier should be treated as a relative or absolute path.
+ * @param {string} specifier - The specifier to check.
+ */
+function shouldBeTreatedAsRelativeOrAbsolutePath(specifier) {
+  if (specifier === '') { return false; }
+  if (specifier[0] === '/') { return true; }
```

Comment:
Shouldn't this be path.sep, since Windows have a different path separator?

Reply:
I moved this so I could re-use it in `import_map.js`, it is just as it was before I moved it. I am thinking maybe this is because imports/require statements are all using nix style, but honestly I am not certain without digging around a bit.